CONTRIBUTING.md

```diff
@@ -84,6 +84,20 @@ Go ahead and push your changes to GitHub using this command.
 git push
 ```
 
+## Cherry-pick PRs to release branches
+
+For example, to cherry-pick a merged PR to release-1.7 branch, on the local release-1.7 branch created off upstream/release-1.7, run:
```

Comment:
Can you please also add how to create the local branch from remote upstream?

Reply:
Done